src/main/java/net/ccbluex/liquidbounce/injection/forge/mixins/entity/MixinEntityLivingBase.java

```diff
@@ -170,4 +172,14 @@ private float hookBodyRotationsB(EntityLivingBase instance) {
 
         return instance instanceof EntityPlayerSP && Rotations.INSTANCE.shouldUseRealisticMode() && rotation != null ? rotation.getYaw() : instance.rotationYaw;
     }
+
+    /**
+     * @author SuperSkidder
+     * @reason Animations swing speed
+     */
+    @Overwrite
+    public int getArmSwingAnimationEnd(){
+        int speed = ModuleManager.INSTANCE.getModule(Animations.class).getState() ? (int) (2 + (20 - Animations.INSTANCE.getSwingSpeed())) : 6;
```

Comment:
Here is a question about `(int) (2 + (20 - Animations.INSTANCE.getSwingSpeed()))` part. What's the point of making Animations.swingSpeed a FloatValue and casting the value to int in this case? It will just cut the fractional part off.
It's better to change the data type of Animations.swingSpeed to IntegerValue, isn't it?

Reply:
Yes my fault